website/pages/docs/enterprise/lease-count-quotas/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+page_title: Vault Enterprise Lease Count Quotas
+sidebar_title: Lease Count Quotas
+description: |-
+  Vault Enterprise features a mechanism to create lease count quotas.
+---
+
+# Lease Count Quotas
+
+Vault features an extension to resource quotas that allows operators to enforce
+limits on how many leases are created. For a given lease count quota, if the
+number of leases in the cluster hits the configured limit, `max_leases`, additional
+lease creations will be forbidden for all clients until a lease has been revoked
+or has expired.
+
+All the nodes in the Vault cluster will share the lease quota rules, meaning that
+the lease counters will be shared, regardless of which node in the Vault cluster
+receives lease generation requests. Lease quotas can be imposed across Vault's API,
+or scoped down to API pertaining to specific namespaces or specific mounts.
+
+Lease count quotas defined in a namespace will be inherited by all the child
+namespaces. By extension, this means that a quota that is defined in the `root`
+namespace is inherited by all namespaces and mounts, essentially to the entire
+Vault API.
+
+Lease count quotas defined on a namespace will take precedence over the inherited
```

Comment:
This block is applicable to both rate and lease quotas. We may need to put this under something like `Quota Prececences` and replicate it rate quota page as well.

Reply:
@vishalnayak there is already a similar section for rate limit quotas